aries_vcx/src/core/profile/mixed_breed_profile.rs

```diff
@@ -0,0 +1,44 @@
+use std::sync::Arc;
+
+use aries_vcx_core::{
+    anoncreds::{base_anoncreds::BaseAnonCreds, credx_anoncreds::IndyCredxAnonCreds, indy_anoncreds::IndySdkAnonCreds},
+    ledger::{base_ledger::BaseLedger, indy_ledger::IndySdkLedger},
+    wallet::{base_wallet::BaseWallet, indy_wallet::IndySdkWallet},
+    PoolHandle, WalletHandle,
+};
+
+use super::profile::Profile;
+
+#[derive(Debug)]
+pub struct MixedBreedProfile {
+    wallet: Arc<dyn BaseWallet>,
+    ledger: Arc<dyn BaseLedger>,
+    anoncreds: Arc<dyn BaseAnonCreds>,
+}
+
+impl MixedBreedProfile {
+    pub fn new(indy_wallet_handle: WalletHandle, indy_pool_handle: PoolHandle) -> Self {
+        let wallet = Arc::new(IndySdkWallet::new(indy_wallet_handle));
+        let ledger = Arc::new(IndySdkLedger::new(indy_wallet_handle, indy_pool_handle));
```

Comment:
Why create a new profile? The proxy profile could have been used with `IndySdkAnoncreds` swapped for `IndyCredxAnonCreds`.

Reply:
Because @Patrik-Stas said he'd like to have cross-libraries testing and it would be clear to remove this if it is it's own individual type rather than tweaking some other type. And I really wanted to use that name somewhere 🤣 . But it will go away soon.

Reply:
I wasn't thinking in particular of trying this combination, why not. It's good to know this also works :-) 
I agree we can remove it in some subsequent PR.